specification/timeseriesinsights/resource-manager/readme.python.md

```diff
@@ -0,0 +1,28 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.timeseriesinsights
+  package-name: azure-mgmt-timeseriesinsights
+  package-version: 0.1.0
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/timeseriesinsights/azure-mgmt-timeseriesinsights/azure/mgmt/timeseriesinsights
```

Comment:
The output folder path looks verbose and duplicated, can this be simplified?

Reply:
that's a standard for python sdk....